opencti-platform/opencti-front/src/private/components/threats/intrusion_sets/Root.tsx

```diff
@@ -224,10 +214,10 @@ const RootIntrusionSet = ({ intrusionSetId, queryRef }: RootIntrusionSetProps) =
               <Route
                 path="/knowledge/*"
                 element={
-                  <div data-testid="instrusionSet-knowledge" key={forceUpdate}>
+                  <div key={forceUpdate}>
```

Comment:
we don't need anymore the data-testid instrusionSet-knowledge ?

Reply:
i added this few weeks ago but i don't need it anymore